velox/exec/tests/RowContainerTest.cpp

```diff
@@ -2063,3 +2063,67 @@ TEST_F(RowContainerTest, hugeIntStoreWithNulls) {
   data->extractColumn(rows.data(), kNumRows, kColumnIndex, extracted);
   assertEqualVectors(source, extracted);
 }
+
+TEST_F(RowContainerTest, columnHasNulls) {
+  auto rowContainer =
+      makeRowContainer({BIGINT(), BIGINT()}, {BIGINT(), BIGINT()}, false);
+  for (int i = 0; i < rowContainer->columnTypes().size(); ++i) {
+    ASSERT_TRUE(!rowContainer->columnHasNulls(i));
+  }
+
+  const uint64_t kNumRows = 1000;
+  VectorFuzzer fuzzer({.vectorSize = kNumRows, .nullRatio = 0}, pool());
+
+  VectorFuzzer fuzzerWithNulls(
+      {.vectorSize = kNumRows, .nullRatio = 0.01}, pool());
+
+  std::vector<VectorPtr> children;
+  const RowTypePtr& rowType = ROW({BIGINT(), BIGINT(), BIGINT(), BIGINT()});
+  children.reserve(rowType->size());
+  for (auto i = 0; i < rowType->size(); ++i) {
+    if (i % 2 == 0) {
+      children.emplace_back(fuzzer.fuzzFlat(rowType->childAt(i), kNumRows));
+    } else {
+      children.emplace_back(
+          fuzzerWithNulls.fuzzFlat(rowType->childAt(i), kNumRows));
```

Comment:
There is a slight chance this will not produce any nulls making the test flaky.

Reply:
I see, updated, thanks.